### ISSUE TITLE
High variance and magnitude of val_rpn_bbox_loss

### ISSUE THREAD

**Issue by bieniaslukasz** [matterport/Mask_RCNN#1243]
I have trained provided model on Balloon dataset (pretrained weights of ImageNet) as well as on COCO dataset (pretrained weights of ImageNet), with provided configurations and training stages. I am observing **high variance** and **high magnitude** of validation loss for RPN bounding box, while the training loss of RPN bounding box is stable and slowly decreases. Is it a normal behaviour? It strongly influence tha validation loss, which as consequence varies a lot also. Did anybody got the same results after training provided Balloon model?

Balloon Training:
![image](https://user-images.githubusercontent.com/32669247/51478467-d1430200-1d8b-11e9-92c7-4d42391e05da.png)

![image](https://user-images.githubusercontent.com/32669247/51478490-e61f9580-1d8b-11e9-95b1-b8d56e617b59.png)

![image](https://user-images.githubusercontent.com/32669247/51478528-f9cafc00-1d8b-11e9-973b-5144d1ba7ec4.png)

![image](https://user-images.githubusercontent.com/32669247/51478626-4282b500-1d8c-11e9-865b-3e751226d947.png)

Thank you in advance for any tips!

Regards,
Lukasz

**Comment by lgreffier** [comment]
I have the same issue. I work with 5 classes + BG. I changed lots of parameters but my VAL_RPN_BBOX_LOSS and VAL_LOSS stay stable with high variance.
Does anyone have an idea for this issue ?

Thanks you for your propositions that could help us !
